### PR TITLE
added keyboard nav to onboarding popups

### PR DESCRIPTION
**Pull request by dnahol**
Addresses Issue #1538 

This patch allows user to navigate through onboarding panels (e.g. Get Started) via pressing "return" on the keyboard.

#OutreachyApplicant

### CODE REVIEW

**Comment by dnahol** [comment]
Hi @groovecoder and @maxxcrawford. Here is a feature I wanted while working with the container popup panels. Let me know what you think!

**Comment by dnahol** [comment]
Hi @groovecoder and @maxxcrawford,

Happy Monday!

Just following up again. Hoping to get a review/approval on this. (Outreachy applicant)

File: src/js/popup.js [hunk 1]
```diff
@@ -382,6 +382,11 @@ const Logic = {
       }
     }
   },
+
+  getCurrentPanel() {
```
Comment (groovecoder) [inline src/js/popup.js]: Can we call this `getCurrentPanelElement` to help distinguish it from `_currentPanel` property and the `getPanelSelector` function already defined on `this`?

Comment (dnahol) [reply src/js/popup.js]: Oh yes, good point! I just changed it! Thank you!

**Comment by groovecoder** [comment]
Dang, now there's a small conflict to resolve too. Sorry. 😢

**Comment by dnahol** [comment]
Merged the conflicts. Checked locally. Everything works.

**Pull request merged**